prow/jobs/jobs.yaml

```diff
@@ -6,4 +6,272 @@ periodics:
     serviceAccountName: pre-submit-service-account
     containers:
     - image: alpine
-      command: ["/bin/date"]
\ No newline at end of file
+      command: ["/bin/date"]
+
+postsubmits:
+  #TODO(vijat@): Find a programmatic way to generate postsubmit jobs for all the service controllers
+  aws-controllers-k8s/sagemaker-controller:
+  - name: sagemaker-post-submit-release
+    decorate: true
+    extra_refs:
+    - org: aws-controllers-k8s
+      repo: test-infra
+      base_ref: main
+      workdir: true
+    spec:
+      serviceAccountName: post-submit-service-account
+      containers:
+        - image: errorbyte/ack-prow-cd #TODO(vijat@): find the correct place for hosting this image.
+          securityContext:
+            privileged: true
+          command: ["/bin/bash", "-c", "cd cd/scripts && ./release-controller.sh"]
```

Comment:
Is this not the default entrypoint command for the containers? Why do we need to keep using this?

Reply:
Prow mandates that podspec mentions the command and not use container image's default entrypoint.